src/main/resources/io/jenkins/plugins/designlibrary/Localization/index.properties

```diff
@@ -5,3 +5,10 @@ titlejelly=Localize text in Jelly
 howtojelly=Use <em>$&#123;%Text}</em> to create texts that can be localized. Create a properties files for content longer than a few words.
 longtext=A longer text that should not appear inline in the jelly file.
 leadParagraph=A detailed explanation how to localize texts in Java and Jelly files can be found in the jenkins.io developer guide <a href="https://www.jenkins.io/doc/developer/internationalization/" target="_blank">Internationalization and Localization</a>.
+
+titlejs=Localize text in JavaScript
+howtojs-jelly-title=Via Jelly
```

Comment:
normally use camel case in jelly howtojsJellyDescription

kebab case looks quite weird in jelly